src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1281,4 +1281,16 @@
   <data name="CollectionsShouldImplementGenericInterfaceMultipleMessage" xml:space="preserve">
     <value>Type '{0}' directly or indirectly inherits '{1}' without implementing any of '{2}'. Publicly-visible types should implement the generic version to broaden usability.</value>
   </data>
+  <data name="MakeTypesInternalCodeFixTitle" xml:space="preserve">
+    <value>Make all public types internal</value>
```

Comment:
I believe it will not fix all public types, also wonder if we should remove the `public` too 
```suggestion
    <value>Make the public type internal</value>
```